<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecDialog</class>
 <widget class="QDialog" name="SecDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>682</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>100</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Main Window</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>You might be interested</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="4">
      <widget class="QListWidget" name="listWidget_defaultList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>146</red>
             <green>194</green>
             <blue>242</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="HighlightedText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>146</red>
             <green>194</green>
             <blue>242</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="HighlightedText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="HighlightedText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Who has &quot;decision-authority&quot; about paper content: First author or the co-author with the higher academic rank?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>How to discuss traumatic experiences that might sound fabricated or distorted in graduate school applications</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>What is the position of Lebanon about the presence of the Hezbollah on its territory?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Is there any evidence on whether &quot;Balrog&quot; from Street Fighter is named after the Balrogs in The Lord of the Rings?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Does God the Son have Faith?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Reference Request for global HÃ¶lder continuity of solutions to elliptic PDEs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Draw a string as a binary tree</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Disk-sound after mounting</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Is it possible to have two normally open circuits and use one switch to close one or the other (never both at once for my purposes)?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Covering directions with rational circles</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>How do I justify a society where people have two permanent homes: a summer home and a winter home?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Why do communists claim that capitalism began at the Industrial Revolution?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A short series of books in which the main character belongs to a race of beings who can create/reset the universe</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Why are sportsmen not desirous to have others of their persuasion move in?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>What do the Apollo crew mean by &quot;you're five square&quot;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Prepping basement for drywall; how to deal with electrical wires running across joists?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>In a perfect vacuum, shouldn't every solid be above its sublimation point, since its vapor pressure must exceed the atmospheric pressure?</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>From Favorite Communities</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QListWidget" name="listWidget_favoriteList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>146</red>
             <green>194</green>
             <blue>242</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="HighlightedText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>146</red>
             <green>194</green>
             <blue>242</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="HighlightedText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="HighlightedText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>[Coding] Draw a string as a binary tree</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>[Cryptography] Explain eliptic curve cryptography like I am five years old?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>[Cryptography] What is symmetrical and assymetrical encryption?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>[Math] Prove 1=2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>[Science] Where does most of Earth's oxygen come from?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>[Science] In a perfect vacuum, shouldn't every solid be above its sublimation point, since its vapor pressure must exceed the atmospheric pressure?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>[Politics] Why do communists claim that capitalism began at the Industrial Revolution?</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>[Religion] Does God the Son have faith?</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="4" rowspan="3" colspan="2">
      <widget class="QListWidget" name="listWidget_communitiesList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>146</red>
             <green>194</green>
             <blue>242</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="HighlightedText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>146</red>
             <green>194</green>
             <blue>242</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="HighlightedText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="HighlightedText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Animals</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Architecture</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Astronomy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Coding</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Computer Hardware</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Computer Science</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cryptography</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cycling</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dating</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Drinks</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Economy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Engagement</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Engineering</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Finances</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Food</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Furniture</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Housing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jobs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Marriage</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Math</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Parenting</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Technology</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Traveling</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Politics</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Psychology</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Religion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Science</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0" colspan="6">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_askAQuestion">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>190</green>
               <blue>111</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>190</green>
               <blue>111</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>190</green>
               <blue>111</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ask a Question</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_searchBar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_searchButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>143</red>
               <green>240</green>
               <blue>164</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>143</red>
               <green>240</green>
               <blue>164</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>143</red>
               <green>240</green>
               <blue>164</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_profileButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="4" colspan="2">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Communities</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
